Assets/GothicVR/Scripts/SO Template/DateTimeGameEvent.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using UnityEngine;
+using UnityEngine.Events;
+
+[CreateAssetMenu(menuName = "Events/DateTime Event Channel")]
```

Comment:
Just checking: Is "channel" the right wording? Or would "Event"/"Callback"/etc. be a better name?